_core/smart/neon-node.md

```diff
@@ -27,64 +27,57 @@ git clone https://github.com/blockstack/stacks-blockchain.git
 cd stacks-blockchain
 ```
 
-Then build the project:
+### Configure your node to connect to the Neon network
 
-```bash
-cargo build
-```
+You'll need to update your node's configuration to connect to the public Neon testnet.
 
-Building the project on ARM:
-
-```bash
-cargo build --features "aarch64" --no-default-features
-```
+In the `stacks-blockchain` repository, open up the file `testnet/follower-config.toml`.
 
-### Setup your Proof-of-Burn miner
+First, find the `bootstrap_node` line in the `[node]` section, and update it to the following:
 
-In this phase of the testnet, miners will be burning BTC to participate in leader election. In order to mine, you'll need to configure your node with a BTC keychain, and you'll need to fund it with BTC. We're providing a BTC faucet to make it easy to run a miner.
+```toml
+bootstrap_node = "048dd4f26101715853533dee005f0915375854fd5be73405f679c1917a5d4d16aaaf3c4c0d7a9c132a36b8c5fe1287f07dad8c910174d789eb24bdfb5ae26f5f27@35.245.47.179:20444"
+```
 
-Let's start by generating a keypair:
+Next, find the `peer_host` line in the `[burnchain]` section, and update it to:
```

Comment:
might be a thing people will mix up, there are actually two `peer_host` lines - one of which is commented out (but still in the burnchain section). Would call that out be avoid issues